public/logos/1850/SOO3.svg

```diff
@@ -0,0 +1,98 @@
+   id="path16" />
+	<svg:path
+   class="st0"
+   d="m 354.69,380.41 c -4.68,8.59 -2.65,17.51 -3.02,26.11 -0.69,15.73 -0.61,31.5 -0.89,47.25 -0.3,16.97 -0.54,33.94 -1.02,50.91 -0.24,8.4 -0.2,16.81 -0.5,25.21 -0.08,2.22 0.41,3.93 1.94,5.47 0.59,0.6 1.38,1.16 1.52,2.35 -12.22,1.92 -24.41,3.84 -36.68,5.77 -0.22,-1.33 0.35,-2.28 0.77,-3.16 1.94,-4.08 2.28,-8.36 2.34,-12.83 0.23,-20.07 0.65,-40.15 1.02,-60.22 0.31,-16.56 0.65,-33.13 0.97,-49.69 0.14,-7.02 0.23,-14.03 0.42,-21.05 0.1,-3.66 0.74,-7.4 -3.3,-10.38 12.44,-1.96 24.18,-3.81 36.43,-5.74 z"
+   id="path18" />
+	<svg:path
+   d="m 215.17,387.9 c 20.1,12.89 36.29,28.31 40.84,52.44 3.78,20.05 2.86,39.84 -6.55,58.44 -6.63,13.1 -17.42,21.25 -31.65,24.91 -2.17,0.56 -2.69,0.2 -2.68,-2.1 0.08,-26.13 0.05,-52.26 0.05,-78.39 0,-17.39 0,-34.79 0,-52.18 -0.01,-0.89 -0.01,-1.77 -0.01,-3.12 z"
+   id="path20" />
+	<svg:path
+   d="m 182.12,324.95 c -6.69,-4.84 -12.77,-9.87 -17.93,-15.94 -5.85,-6.89 -8.92,-14.93 -9.6,-23.87 -1.11,-14.44 -0.45,-28.7 5.26,-42.29 4.45,-10.59 11.43,-18.9 22.27,-24.01 0,35.38 0,70.57 0,106.11 z"
+   id="path22" />
+	<svg:path
+   d="m 571.09,221.03 c 0.13,15 -2.29,28.16 -9.31,40.21 -5.15,8.83 -12.5,14.4 -22.94,15.53 -10.02,1.08 -17.84,-2.7 -23.86,-10.5 -5.67,-7.35 -8.36,-15.97 -10.17,-24.89 -2.46,-12.14 -3.38,-24.4 -1.95,-36.73 1.22,-10.53 3.89,-20.65 9.72,-29.69 6.87,-10.65 18.41,-15.71 30.77,-13.5 5.96,1.06 10.36,4.56 13.89,9.28 6.95,9.31 10.44,19.99 12.26,31.31 1.06,6.69 1.77,13.41 1.59,18.98 z"
+   id="path24" />
+	<svg:path
+   d="m 408.87,246.75 c 0.15,14.68 -2.26,27.69 -9.09,39.62 -5.11,8.93 -12.45,14.68 -22.99,15.86 -10.23,1.15 -18.15,-2.74 -24.23,-10.75 -5.29,-6.97 -7.98,-15.09 -9.72,-23.52 -3.2,-15.47 -4.12,-31.02 -0.74,-46.6 1.81,-8.34 4.55,-16.37 9.88,-23.19 7.6,-9.72 17.7,-13.25 29.71,-11.13 5.65,1 9.86,4.53 13.23,9.03 7.01,9.37 10.54,20.12 12.36,31.53 1.07,6.71 1.78,13.43 1.59,19.15 z"
+   id="path26" />
+</svg:g>
+<script
+   id="bw-fido2-page-script" /></svg:svg>
```

Comment:
GitHub is saying this image can't render due to an error, and the closing svg tags here are highlighted in red when viewing in text mode. I think these last 3 lines just need to be deleted?

Otherwise, looks good.

Reply:
That file was accidentally included.  I removed it, and the other SOO.svg file should be correct.